pkg/util/containers/v2/metrics/collector_ecs_fargate.go

```diff
@@ -0,0 +1,193 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build docker
```

Comment:
Is this required due to the `pkg/util/ecs/metadata` deps?

Reply:
yes